README.md

```diff
@@ -184,6 +185,7 @@ Request parameters:
 - `showTips` - (optional) whether to show (true) or hide (false) the tip options. Default is (false)
 - `payment_method` - (optional) Settings to enable going straight to QR scanning
   - `payment_method_type` - Payment method type: CARD (default), ALIPAY, WECHAT
```

Comment:
Unrelated, but are these two really snake_case? All the rest are CamelCase. 🤔

Reply:
Good point! They should all be camel case... good catch!

Reply:
Fixed...